zerver/views/scheduled_messages.py

```diff
@@ -23,3 +26,40 @@ def get_scheduled_message(scheduled_message_id: int, user_profile: UserProfile)
         raise ResourceNotFoundError(_("Scheduled message does not exist"))
 
 
+def fetch_scheduled_messages(request: HttpRequest, user_profile: UserProfile) -> HttpResponse:
+    scheduled_messages = ScheduledMessage.objects.filter(
+        sender=user_profile, delivered=False, delivery_type=ScheduledMessage.SEND_LATER
+    )
+    scheduled_messages_to_dict: List[ScheduledMessageDict] = []
+
+    for msg in scheduled_messages:
+        to, _type = get_recipient_ids(msg.recipient, user_profile.id)
```

Comment:
Let's use `for scheduled_message in scheduled_messages:` as our loop structure, and `recipient, recipient_type_str = get_recipient_ids(...)` -- that fits our patterns for how we name local variables much better.

Reply:
Okay, makes sense.